docs/api/materials/Material.html

```diff
@@ -162,8 +162,8 @@ <h3>[property:String name]</h3>
 
 		<h3>[property:Boolean needsUpdate]</h3>
 		<p>
-		Specifies that the material needs to be updated at the WebGL level.
-		Set it to true if you made changes that need to be reflected in WebGL.<br />
+		Specifies that the material needs to be recompiled.
+		Set it to true if you want the shader to be recompiled.<br />
```

Comment:
redundant, but about as useful as the previous sentence, just bouncing ideas

Reply:
Yes, it's redundant. You can remove this sentence.